### PR TITLE
Rounded corners on the chart and styled grid line

### PR DESCRIPTION
**Pull request by bhargavigundaa**
Fixed review comments, review and then merge

### CODE REVIEW

File: learn/app-development/widgets/chart/chart-widget.md [hunk 1]
```diff
@@ -230,6 +230,24 @@ Page.chart4_1Beforerender = function(widget, chartInstance) {
 
 [![](/learn/assets/onbeforerender-code.png)](/learn/assets/onbeforerender-code.png)
 
+### Customizing charts with rounded corners
+Inorder to achieve rounded corners for column and bar charts add class **'rounded-cornered-chart'** and for area chart add class **'rounded-area-chart'** to the chart widget.
+
+#### Column Chart
+[![](/learn/assets/rounded-column-chart.png)](/learn/assets/rounded-column-chart.png)
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/chart/chart-widget.md:237]: Can we fix the rounded edge so it is not so rounded?

File: website/static/learn/assets/rounded-column-chart.png
Comment (prasanth-reddy-wavemaker-com) [inline website/static/learn/assets/rounded-column-chart.png:1]: Too much roundedness. Can we make this look better?

File: website/static/learn/assets/rounded-bar-chart.png
Comment (prasanth-reddy-wavemaker-com) [inline website/static/learn/assets/rounded-bar-chart.png:1]: Same, too much roundedness.

File: learn/app-development/widgets/chart/chart-widget.md [hunk 2]
```diff
@@ -230,6 +230,24 @@ Page.chart4_1Beforerender = function(widget, chartInstance) {
...
+Inorder to achieve rounded corners for column and bar charts add class **'rounded-cornered-chart'** and for area chart add class **'rounded-area-chart'** to the chart widget.
+
+#### Column Chart
+[![](/learn/assets/rounded-column-chart.png)](/learn/assets/rounded-column-chart.png)
+
+#### Bar Chart
+[![](/learn/assets/rounded-bar-chart.png)](/learn/assets/rounded-bar-chart.png)
+
+### Customizing grid line on the charts
+Solid stroked line on x-axis and y-axis zero line can be achieved by adding **'stroked-zero-axis-line'** class
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/chart/chart-widget.md:243]: Are these CSS classes available in nvd3? IF so, can we just show a list all available CSS classes and the effect adding them will have?

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]

File: learn/app-development/widgets/chart/chart-widget.md (hunk 1 above)
Comment (bhargavigundaa) [reply learn/app-development/widgets/chart/chart-widget.md:237]: Sure, will fix it in upcoming releases and make changes in the pr accordingly.

File: learn/app-development/widgets/chart/chart-widget.md (hunk 2 above)
Comment (bhargavigundaa) [reply learn/app-development/widgets/chart/chart-widget.md:243]: These are the classes written by us only on the platform using nvd3 selectors, like below

.stroked-zero-axis-line {
      .nvd3 .nv-axis .zero line {
        stroke: rgba(24, 21, 21, 1);
        stroke-dasharray: none; 
      }
    }

File: learn/app-development/widgets/chart/chart-widget.md (hunk 1 above)
Comment (bhargavigundaa) [reply learn/app-development/widgets/chart/chart-widget.md:237]: Screenshot 2023-08-29 at 12 50 23 PM

Screenshot 2023-08-29 at 12 50 05 PM

Comment (bhargavigundaa) [reply learn/app-development/widgets/chart/chart-widget.md:237]: We can adjust the property values in css and can achieve these as well

File: website/static/learn/assets/rounded-bar-chart.png
Comment (bhargavigundaa) [reply website/static/learn/assets/rounded-bar-chart.png:1]: This is updated

File: website/static/learn/assets/rounded-column-chart.png
Comment (bhargavigundaa) [reply website/static/learn/assets/rounded-column-chart.png:1]: @prasanth-reddy-wavemaker-com : Please check the updated images

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]

**Pull request merged**